test/run_unit_tests.sh

```diff
@@ -69,21 +69,21 @@ run_doctest cime_config/atm_in_paramgen.py
 # CAM history config doctests:
```

Comment:
Can we rename this file to `run_python_unit_tests.sh`?  I realize we don't want this long-term but hopefully it will help avoid any confusion in the interim.  Thanks!